src/super_gradients/examples/yolov5_transfer_learning_example/yolov5_transfer_learning_example.py

```diff
@@ -1,102 +0,0 @@
-"""
```

Comment:
don't you prefer fixing this with Yolox?

Reply:
its a bad idea to keep them inside the repo IMO.